RaveBusinessLogic/BRAT.xml

```diff
@@ -9,28 +9,28 @@
         <Children collapsed="true">
           <Node xpathlabel="Name" xpath="SQLiteDB" type="file" />
           <Node xpathlabel="Name" xpath="HTMLFile" type="report" />
-          <Node label="Capacity" id="capacity">
+          <Node label="Capacity">
             <Children>
-              <Node label="Existing Dam Capacity" xpath="Vector" type="line" symbology="oCC_EX" />
+              <Node label="Existing Dam Capacity" xpath="Vector" type="line" symbology="oCC_EX" id="ex_capacity" />
               <Node label="Existing Dam Size" xpath="Vector" type="line" symbology="mCC_EX_CT" />
-              <Node label="Historic Dam Capacity" xpath="Vector" type="line" symbology="oCC_HPE" />
+              <Node label="Historic Dam Capacity" xpath="Vector" type="line" symbology="oCC_HPE" id="his_capacity" />
               <Node label="Historic Dam Size" xpath="Vector" type="line" symbology="mCC_HPE_CT" />
 
-              <Node label="Existing Dam Capacity" xpath="Geopackage/Layers/Vector[@id='BRAT_RESULTS']" type="line" symbology="oCC_EX" />
+              <Node label="Existing Dam Capacity" xpath="Geopackage/Layers/Vector[@id='BRAT_RESULTS']" type="line" symbology="oCC_EX" id="ex_capacity" />
               <Node label="Existing Dam Size" xpath="Geopackage/Layers/Vector[@id='BRAT_RESULTS']" type="line" symbology="mCC_EX_CT" />
-              <Node label="Historic Dam Capacity" xpath="Geopackage/Layers/Vector[@id='BRAT_RESULTS']" type="line" symbology="oCC_HPE" />
+              <Node label="Historic Dam Capacity" xpath="Geopackage/Layers/Vector[@id='BRAT_RESULTS']" type="line" symbology="oCC_HPE" id ="his_capacity"/>
               <Node label="Historic Dam Size" xpath="Geopackage/Layers/Vector[@id='BRAT_RESULTS']" type="line" symbology="mCC_HPE_CT" />
             </Children>
           </Node>
           <Node label="Management">
             <Children collapsed="true">
-              <Node label="Unsuitable or Limited Opportunities" xpath="Vector" type="line" symbology="oPBRC_UD" />
-              <Node label="Risk of Undesirable Dams" xpath="Vector" type="line" symbology="oPBRC_UI" />
-              <Node label="Restoration or Conservation Opportunities" xpath="Vector" type="line" symbology="oPBRC_CR" />
+              <Node label="Unsuitable or Limited Opportunities" xpath="Vector" type="line" symbology="oPBRC_UD" id="limited" />
+              <Node label="Risk of Undesirable Dams" xpath="Vector" type="line" symbology="oPBRC_UI" id ="risk" />
+              <Node label="Restoration or Conservation Opportunities" xpath="Vector" type="line" symbology="oPBRC_CR" id="opportunity" />

```

Comment:
These lines seem to be duplicates too. Notice how they double the lines below and how the xpaths all say "vector"  which won't target anything specific enough.

Also same with lines 84-94. Not really sure how this happened but we should remove all the duplicates that don't point to anything since QRAVE will get confused if there are multiple of the same ids in a file,